### ISSUE TITLE
Cannot Edit Photos

### ISSUE THREAD

**Issue by StrangeOne101** [ren7995/Anne#1]
# Description
I can no longer edit or copy photos in the photos app. Whenever you try save an edit or copy a photo, you get a "There was an error saving this photo. Please try again later." message. This makes it impossible to edit photos at all without importing them into another app. Selecting photos in other apps is fine (and when it isn't, a `uicache -a` command alleviates that)

This occurs in normal photos that are not in the photo album, regardless of it's unlocked or hidden state

# Facts
- This happens regardless of whether the hidden album is unlocked or visible (hidden in settings)
- Running `uicache -a` does not resolve the issue - even temporarily
- Rebooting the device and using it in a non-jailbroken state does alleviate the problem
- The hidden album has multiple photos and images saved to it

# Specs
- iPhone 12 Pro
- iOS 14.5
- uncover 8.0.2 jailbreak
- Anne 1.0.1

![64A66FB8-F07D-4269-9B97-94D442311DB0](https://user-images.githubusercontent.com/2769013/178136356-0eb182be-62b7-4351-a0fd-9ff8c1125b21.png)

**Comment by phong010198** [comment]
Same here. Did you find a solution (other from a restore)?

**Comment by StrangeOne101** [comment]
No, not for Anne. In the end, I upgraded iOS which has face ID built into the hidden album in photos

**Comment by phong010198** [comment]
`AnneHelper/main.m` modifies `MobileSlideShow.app/Info.plist`, so I think that is the cause.
For anyone meet this error, restore rootFS -> backup your data -> Erase All Content and Settings -> don't install this tweak till any update.
